From 30fad16bbc29059b4776c47152633c7825b70442 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 12 Sep 2022 17:44:19 +0200
Subject: [PATCH 0331/1695] plat-stm32mp2: conf: default enable STGEN for
 STM32MP2 platforms

Default enable STGEN for STM32MP2 platforms.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ee24358b494c8b79883923e531cf9cb01cd64e5e plat-stm32mp2: conf: default enable STGEN for STM32MP2 platforms]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ia8e2481bbc3f650e69adf963ff9fed0459feeead
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/265963
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 00e73404e..7dc51deb5 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -75,6 +75,7 @@ CFG_STM32_RISAB ?= y
 CFG_STM32_RISAF ?= y
 CFG_STM32_RTC ?= y
 CFG_STM32_SERC ?= y
+CFG_STM32_STGEN ?= y
 CFG_STM32_UART ?= y
 
 # Default enable some test facitilites
-- 
2.47.2

